packages/create-neutrino-project/commands/init/generator/install.js

```diff
@@ -0,0 +1,11 @@
+const commandExists = require('command-exists');
+
+module.exports = function() {
+  const isYarnAvailable = commandExists.sync('yarnpkg');
```

Comment:
Even though the user may have `yarn` installed, that isn't enough to know whether they want it used on this particular project. Instead, you can find out through the environment variables.

```js
const isYarn = process.env.npm_config_user_agent.includes('yarn');
```